server/routes/api/documents/documents.ts

```diff
@@ -97,7 +97,12 @@ router.post(
     };
 
     if (template) {
-      where = { ...where, template: true };
+      where = {
+        ...where,
+        template: true,
+        // override publishedAt filter from Document defaultScope to include drafts.
+        publishedAt: { [Op.or]: { [Op.eq]: null, [Op.ne]: null } },
```

Comment:
I think this will pull other users drafts, also it's a shame that it's so "hacky" of course. It might actually be better to just load drafts from the templates page and wait for both requests to complete. 🤔

Reply:
I wanted to avoid an additional API call; but if that's alright, cool :) Let me update the PR.

Reply:
Understandable, It'll be okay here given it's a settings page.